File: backend/ViewModels/FloorViewModel.cs
using BookingApp.Enums;

namespace BookingApp.ViewModels
{
    public class FloorViewModel : FloorSimpleViewModel
    {
        public List<BookingPlaceWithBookingsViewModel> BookingPlaces { get; set; } = new List<BookingPlaceWithBookingsViewModel>();
    }
}

File: frontend/src/app/api-generated/models/floor-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { BookingPlaceWithBookingsViewModel } from '../models/booking-place-with-bookings-view-model';
export interface FloorViewModel {
  bookingPlaces?: Array<BookingPlaceWithBookingsViewModel> | null;
  description?: string | null;
  id?: number;
  name?: string | null;
  svg?: string | null;
}
